packages/block-editor/src/components/iframe/index.js

```diff
@@ -167,6 +167,7 @@ function Iframe( {
 		node.addEventListener( 'load', onLoad );
 
 		return () => {
+			delete node._load;
```

Comment:
Why does not doing this prevent the node from being removed from memory? When React removes the iframe, it will remove this callback as well?

This reminds me of https://github.com/WordPress/gutenberg/pull/27544#discussion_r537520633

That said, it doesn't hurt of course, I'd just like to understand why.

Reply:
This is a tricky one that's several levels deep. I'm not entirely sure why this fix helps either! I'll try to post more details after I have a clearer vision.

Reply:
Okay, I still don't fully understand what happened. It seems to require a very specific setting to make it reproducible. For instance, there has to be a `.focus()` call on a `<input>` somewhere, the iframe has to be rendered under a condition and a wrapper, etc. This might also only be reproducible in a chromium-based browser.

However, we can still learn from this mistake and prevent it from happening again by following the general best practice: don't cross-reference between parent and child frames. In particular, the `window.frameElement` call is the code smell we want to avoid. Instead, use [`postMessage`](https://web.dev/detached-window-memory-leaks/#solution-postmessage) to communicate to the parent frame. In this case though, deleting `node._load` has the same effect, and it's easier than refactoring to `postMessage`.